### PR TITLE
feat(ArtistAbout): fix styling in some links and read more on bio

### PR DESCRIPTION
**Pull request by araujobarret**
### Description

This PR adjusts some inconsistency around each section "View All" and adds `...` to the Read More truncated text.

Screenshot 2023-07-21 at 11 11 39

@artsy/diamond-devs 

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- styles changes in Artist About sections - araujobarret

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ❌ **An error occurred while validating your changelog, please make sure you provided a valid changelog.**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 2abcf4ade3b3cee38b1019da39ea7887d9ccb4e6
</p>

File: src/app/Components/Artist/Biography.tsx [hunk 1]
```diff
@@ -25,13 +25,15 @@ export const Biography: React.FC<BiographyProps> = ({ artist }) => {
...
+      <Text>
+        <Text>{`${expanded ? text : truncatedText}${
+          text.length > MAX_CHARS && !expanded ? "... " : " "
+        }`}</Text>
 
         {!!canExpand && (
-          <Touchable onPress={() => setExpanded((e) => !e)} haptic>
-            <Text underline>{expanded ? "Read Less" : "Read More"}</Text>
-          </Touchable>
+          <Text underline onPress={() => setExpanded((e) => !e)}>
```
Comment (jo-rs) [inline src/app/Components/Artist/Biography.tsx:34]: It feels a little weird to me to have a `Text` have an `onPress` prop, but after discussing, I understand that this is a temporary solution which will be eventually addressed in a more general "Read More" Palette component.

**Review by jo-rs — APPROVED** [review]
Looks good! 
I left a comment with my only issue for the sake of visibility, but we already discussed this in private 👍

File: src/app/Components/Artist/Biography.tsx (hunk 1 above)
Comment (araujobarret) [reply src/app/Components/Artist/Biography.tsx:34]: Yeah, we're gonna discuss if we use `Touchable` or the native onPress from `Text` in Palette, gonna open a PR there soon with the `ReadMore` element 👍 
https://reactnative.dev/docs/text#onpress

**Pull request merged**